ci/release/update-version.sh

```diff
@@ -56,3 +56,14 @@ sed_runner "/TAGFILES/ s|[0-9]\+.[0-9]\+|${NEXT_SHORT_TAG}|g" cpp/doxygen/Doxyfi
 for FILE in .github/workflows/*.yaml; do
   sed_runner "/shared-action-workflows/ s/@.*/@branch-${NEXT_SHORT_TAG}/g" "${FILE}"
 done
+
+# Need to distutils-normalize the original version
+CURRENT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${CURRENT_SHORT_TAG}'))")
+NEXT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${NEXT_SHORT_TAG}'))")
+
+# Dependency versions in dependencies.yaml
+sed_runner "s/==${CURRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" dependencies.yaml
+# Dependency versions in setup.py
+sed_runner "s/==${CURRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" python/cuspatial/setup.py
+# Dependency versions in pyproject.toml
+sed_runner "s/==${CURRENT_SHORT_TAG_PEP440}.*\"/==${NEXT_SHORT_TAG_PEP440}.*\"/g" python/cuspatial/pyproject.toml
```

Comment:
```suggestion
# Need to distutils-normalize the original version
NEXT_SHORT_TAG_PEP440=$(python -c "from setuptools.extern import packaging; print(packaging.version.Version('${NEXT_SHORT_TAG}'))")

# Dependency versions in dependencies.yaml
sed_runner "/-cu11==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" dependencies.yaml
```

This change will work. It doesn't look like any relevant dependencies currently exist in `python/cuspatial/setup.py` or `python/cuspatial/pyproject.toml`, so I've removed those lines.

And I tweaked the `sed` command a bit for `dependencies.yaml`.

FWIW, Vyas and I are planning on overhauling `update-version.sh` in the future.

Among other improvements, we will be removing the `sed_runner` shim here so that we can use the full set of flags/features provided by `sed`.

When that happens, I will go back and simplify/dedupe some of these replacement commands throughout the repositories.

Reply:
Updated the PR suggestion to use: `sed_runner "/-cu[0-9]\{2\}==/ s/==.*/==${NEXT_SHORT_TAG_PEP440}.*/g" dependencies.yaml`, which will match any `-cuDD` string, where `D` is any digit.